src/odemis/model/_metadata.py

```diff
@@ -163,6 +163,7 @@
 MD_ROTATION_COR = "Rotation cor" # radians, to be subtracted from MD_ROTATION
 MD_PIXEL_SIZE_COR = "Pixel size cor" # (m, m), to be multiplied with MD_PIXEL_SIZE
 MD_POS_COR = "Centre position cor"  # (m, m), to be subtracted from MD_POS
+MD_CALIB = "Calibration parameters"  # (list of list of float) Calibration parameters for the correct axes mapping
```

Comment:
Please move this next to `MD_SHIFT_LOOKUP`. As comment, let's keep it generic: "Hardware-specific calibration parameters". Hopefully we can reuse this metadata for a couple of extra drivers ;-)